config/client.lua

```diff
@@ -0,0 +1,53 @@
+return {
+    useTarget = GetConvar('UseTarget', 'false') == 'true',
```

Comment:
We actually don't have a convar for targeting. We've decided to go the route of per-resource usage so this can just be defaulted to false.